### ISSUE TITLE
No window content drawn when packaging app with PyQt5 >= 5.13.2 on macOS

### ISSUE THREAD

**Issue by SamSchott** [pyinstaller/pyinstaller#4615]
When packing an app written with PyQt5 and using PyQt5 >= 5.13.2, the window content is not drawn and the following error is logged:

```shell
It does not make sense to draw an image when [NSGraphicsContext currentContext] is nil. 
This is a programming error. Break on void _NSWarnForDrawingImageWithNoCurrentContext(void)
to debug.  This will be logged only once.  This may break in the future.
```

This issue does not affect PyQt5 5.13.1 or lower but has also been reported for PySide2 >= 5.13.2 on [Gitter](https://gitter.im/PySide/pyside2?at=5dcef3f34adf071a843654f6).

System information:

* Platform: macOS 10.15.2 (Catalina)
* Python version: 3.7.6
* PyQt5 version: 5.13.2, 5.14.0
* PyInstaller version: 3.5, 4.0.dev0+a97628fba9

PyInstaller logs have been uploaded to [here](https://gist.github.com/SamSchott/ad4deacc5263f0cbeb8c44db28588517).

**Comment by drinkitup777** [comment]
Do this only occur when you have `NSRequiresAquaSystemAppearance` set to `false` in your Info.plist? If so, then we have [the same issue](https://github.com/pyinstaller/pyinstaller/issues/4627#issue-548593885).

**Comment by SamSchott** [comment]
Yes, I just checked. With `NSRequiresAquaSystemAppearance` set to `True`, everything works well again.

**Comment by MAKOMO** [comment]
While the window content shows up with `NSRequiresAquaSystemAppearance` set to `True` (which seems to be the default as without setting this variable at all it shows the same behaviour) on macOS 10.15, it of course does not respect its dark mode. 

The issue is not be exclusive to pyinstaller as I have experienced the same using py2app. As this has been reported also for PySide2 it might be a bug in Qt (>= 5.13.2) or a change in the Qt builds that causes this. Unclear how to debug this one.

**Comment by crwood** [comment]
My solution to this problem was to recompile both the PyInstaller bootloader and the Python interpreter with a `MACOSX_DEPLOYMENT_TARGET=10.13` (overriding the PyInstaller default of 10.7) and to use Xcode version 11 or higher while doing so. I now have a PyInstaller-frozen (Py)Qt 5.14 application with working dark mode _and_ binary-compatibility with macOS 10.13, 10.14, and 10.15. See [this comment](https://github.com/gridsync/gridsync/issues/267#issuecomment-609980411) for further details.

**Comment by SamSchott** [comment]
Great, thank you for the info!

**Comment by htgoebel** [comment]
Duplicate of #4627
